app/controllers/ems_common.rb

```diff
@@ -201,11 +201,11 @@ def form_field_changed
           # Hide/show C&U credentials tab
           page << "$('#metrics_li').#{params[:server_emstype] == "rhevm" ? "show" : "hide"}();"
         end
-        if ["openstack", "openstack_infra"].include?(params[:server_emstype])
+        if ["openstack", "openstack_infra", "oraclevm"].include?(params[:server_emstype])
```

Comment:
@matthewd @durandom @juliancheal this is a place in the code that needs to ask the model, instead of keeping the knowledge here.